content/en/storage-providers/get-started/tasks/index.md

```diff
@@ -27,6 +23,10 @@ Snap-Deal sealing tasks are covered in a separate dedicated guide in the operate
 
 ![Overview of the lotus-miner tasks](lotus-miner-tasks.png) 
 
+## Sealing tasks
+
+Before a sector can be commited to network, the storage provider *must* seal the sector, meaning it needs to encode the data in the sector to prepare it for the proving tasks. Sealing a sector is a multi-step process and is time-intensive to create, because the encrypted version of each chunk of data depends on every other chunk of input data.
```

Comment:
```suggestion
Before a sector can be committed to the network, the storage provider *must* seal the sector, meaning the data needs to be encoded in the sector to prepare it for the proving tasks. Sealing a sector is a multi-step process and is time-intensive to create because the encrypted version of each chunk of data depends on every other chunk of input data.
```